docs/modules/ROOT/pages/tests-configuration.adoc

```diff
@@ -677,14 +677,33 @@ CAUTION: Use this tool at your own risk, since there is a chance of potential le
 
 === How to use an encrypted string in VIVIDUS
 
-. Add the secret key/text to the project properties as the value of:
+. Set the password in one of the following ways:
++
+.. Pass the key with its password-value into a test run using command line:
++
+[source,properties]
+----
+./gradlew runStories -Pvividus.encryptor.password=MYPAS_WORD
```

Comment:
is it work?
shouldn't it be `-Pvividus.system.vividus.encryptor.password=MYPAS_WORD` ?

Reply:
> is it work? shouldn't it be `-Pvividus.system.vividus.encryptor.password=MYPAS_WORD` ?

It looks like the build system [sets](https://github.com/vividus-framework/vividus-build-system/blob/46acaeeb7806187b90f8238a6ad9ecb229d9bf1b/1.0/gradle/vividus-test.gradle#L133) put all properties with the prefix "vividus." into system properties